### DISCUSSION TITLE
How to make a bigserial field ?

### DISCUSSION THREAD

**Discussion by shahriyardx** [encode/databases#373] [Q&A]
Postgresql supports BIGSERIAL but how to create that in databases postgresql

**Comment by encryptblockr** [comment]
by using `BigInteger`

```
from sqlalchemy import BigInteger

metadata = MetaData()

users = Table(
    "users", metadata,
    Column("id", BigInteger, primary_key=True),
)
```

**Reply by shahriyardx** [reply]
What if I don't want to use it without sqlalchemy. Like writing queries

`id BIGSERIAL PRIMARY KEY`

Can I do this?
